src/components/SearchPanel.vue

```diff
@@ -225,7 +225,7 @@
       },
 
       filterlayers() {
-        return (ApplicationState.tokens.filtertoken && this.state.search_layers.filter(l => l.getFilterToken()) || []);
+        return (ApplicationState.tokens.filtertoken && [].concat(this.state.search_1n_layer || [], this.state.search_layers).filter(l => l.getFilterToken()) || []);
```

Comment:
There are a lot of these statements:

```js
[].concat(state.search_1n_layer || [], state.search_layers)
```

Please try to rewrite it to in order to use only `state.search_layers` and `state.search_1n_relationid`. 

ie. `state.search_1n_layer` is a superfluous property and it just adds complexity.